astropy/tests/plugins/display.py

```diff
@@ -0,0 +1,142 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
Is there any plan to refactor this out to a separate plugin, too? Affiliates are using this, and inheriting the ``PYTEST_HEADER_MODULES`` dictionary with astropy optional dependencies is suboptimal (https://github.com/astropy/pyregion/pull/120#issuecomment-336489498).
However if this getting out of astropy core, I don't try to come up with a workaround for the package template, as then we can keep the default dictionary a rather small one.

Reply:
When @astrofrog and I discussed this (or maybe it was @eteq?) we decided we weren't going to move this in the first pass. I'm pretty sure it should be possible though and seems pretty reasonable. Maybe this doesn't require a separate package but could go in `pytest-astropy`, which is currently just a "meta" package?

Reply:
Yes I think the remaining plugins that aren't of general interest could live in pytest-astropy.